ground-control/internal/server/config_handlers.go

```
@@ -0,0 +1,405 @@
+	}
+
+	tx, err := s.db.BeginTx(r.Context(), nil)
+	if err != nil {
+		log.Println(err)
+		HandleAppError(w, err)
+		return
+	}
+    committed := false
+	defer func() {
+		if !committed {
+			tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)